graylog2-server/src/main/java/org/graylog2/plugin/inputs/transports/AbstractTcpTransport.java

```diff
@@ -142,12 +143,12 @@ public AbstractTcpTransport(
             EventLoopGroup parentEventLoopGroup,
             EventLoopGroupFactory eventLoopGroupFactory,
             NettyTransportConfiguration nettyTransportConfiguration,
-            org.graylog2.Configuration graylogConfiguration) {
+            TLSProtocolsConfiguration tlsConfiguration) {
```

Comment:
Hmm, this is a breaking API change. :thinking:  That means all TCP transports need to be adjusted to work. Also 3rd party ones. I guess we have to add the setting to `org.graylog2.Configuration` as well to avoid this. It's a bit ugly but I don't see any other way, unfortunately. :confused:

Reply:
I've added a backwards compatible constructor